script/bootstrap.js

```diff
@@ -1,139 +1,260 @@
 #! /usr/bin/env node
 
-require('dotenv').config({ path: '.env.local' })
+const { program } = require('commander');
+program.version('0.0.1');
+
+const { google } = require("googleapis");
+const credentials = require("./credentials.json");
+const scopes = ["https://www.googleapis.com/auth/drive"];
+const auth = new google.auth.JWT(credentials.client_email, null, credentials.private_key, scopes);
+const drive = google.drive({ version: "v3", auth });
 
 const shared = require("./shared");
 
+require('dotenv').config({ path: '.env.local' })
+
 const apiUrl = process.env.HASURA_API_URL;
 const apiToken = process.env.ORG_SLUG;
+const adminSecret = process.env.HASURA_ADMIN_SECRET;
 
-async function createGeneralNewsCategory() {
-  const localeResult = await shared.hasuraListLocales({
-    url: apiUrl,
-    orgSlug: apiToken,
-  });
+const metadata = {
+  "shortName": "The New Oaklyn Observer",
```

Comment:
How will we configure this stuff for each org?

Reply:
Good point. I'll think about this today. I thought about doing an interactive CLI for this kind of thing... or we could add these values to the .env.local file (or some other json/data file in the repo)...

Reply:
Actually, I think we should use the tinycms metadata editor, which I believe I'm going to fix up a bit later today as well. This could be the first test run of the frontend for a new tiny news org, I guess.

Reply:
I've updated this PR:

* site metadata no longer touched in this bootstrap command
* also, noticed that google folders were being created even if the organization-create failed; this now only happens if that's successful
* finally, realised we don't need the `script/run` anymore (it's just `yarn dev`) so I renamed it `script/switch` - all it does is change the symlink for your .env file: `script/switch bootstrapper; script/switch oaklyn`